src/pages/explorer.tsx

```diff
@@ -305,7 +305,7 @@ function Page() {
           </Stack>
         </Container>
 
-        {accessList.length > 0 && (
+        { accessList.length >= 0  && (
```

Comment:
I believe this was the bug. Sometimes the access list is 0 when the subsetQuery data returns empty. I don't know the deep details but it seems like you can have valid withdrawals without subset exclusions or inclusions.